mrp_unbuild_advanced/views/mrp_unbuild_views.xml

```diff
@@ -103,4 +103,27 @@
             </field>
         </field>
     </record>
+
+    <record id="mrp_unbuild_calendar_view" model="ir.ui.view">
+        <field name="name">mrp.unbuild.calendar</field>
+        <field name="model">mrp.unbuild</field>
+        <field name="arch" type="xml">
+            <calendar date_start="unbuild_date" quick_add="False" string="Unbuild Orders">
```

Comment:
Depending on web page width, calendar placeholder couldn't be enough

![imagen](https://github.com/solvosci/slv-manufacture/assets/12749832/a5e31f96-a15a-4052-9802-fcfc1449d2f4)

because we have not a `date_delay` or `date_stop` for an unbuild record.

This could be solved e.g. using scheduled_time as date_delay parameter for the view:

```suggestion
            <calendar date_start="unbuild_date" date_delay="scheduled_time" quick_add="False" string="Unbuild Orders">
```

could you check?